src/main/java/com/epam/ms/repository/domain/UserProfile.java

```diff
@@ -0,0 +1,37 @@
+package com.epam.ms.repository.domain;
+
+import lombok.Data;
+import org.hibernate.annotations.GenericGenerator;
+
+import javax.persistence.*;
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "user_profiles", schema = "users")
+@Data
+public class UserProfile {
+    @Id
+    @GeneratedValue(generator = "UUID")
+    @GenericGenerator(name = "UUID", strategy = "org.hibernate.id.UUIDGenerator")
+    private String id;
```

Comment:
may we have one space between two properties, please, to increase readability
```
@Annotation
String a;

@Annotation
String b;
```

Reply:
Thank you, done